xapian-core/tests/testdata/apitest_diversify.txt

```diff
@@ -0,0 +1,21 @@
+The island Java lies between Sumatra to the west and Bali to the East.
```

Comment:
This new file needs to be listed `EXTRA_DIST` in `tests/Makefile.am` or else it won't be included in the tarball build by `make dist` and then someone building a release from a distribution tarball won't be able to run these new tests.

An easy way to test for this (and various other issues) locally is to run `make distcheck` - that makes a distribution tarball, then unpacks it, tries to build it as an out-of-tree build, tries to install it into a temporary location, checks that the testsuite can be run against the installed version (this is a no-op for xapian-core, but omega and the bindings support it), tries to uninstall it, and then checks that the uninstall removed all the files which were installed, and then checks that the tree unpacked from the tarball can also `make dist` successfully.

We don't currently have a build for this on the CI infrastructure that runs for PRs, though perhaps we should.

We do already have coverage for this once code is merged though - there is a cron job which runs `make distcheck` on master and RELEASE/1.4 branches a few times an hour (if there are any changes) and if it works makes a new snapshot available at https://oligarchy.co.uk/xapian/master/ or https://oligarchy.co.uk/xapian/RELEASE/1.4/ depending on which branch it is from.

Reply:
I've noticed ```apitest_cluster.txt``` isn't included in it either, so I believe that should be added too. I've pushed a commit to include ```apitest_diversify.txt``` for now.

Reply:
> I've noticed `apitest_cluster.txt` isn't included in it either, so I think that should be added too.

That one doesn't actually appear to be used anywhere (which would explain why we aren't seeing failures due to it not being included in tarballs).  It looks like its content was inlined into `api_cluster.cc` (with slight tweaks to two of them).

Good spot - I'll push a change to just drop it once I've merged this.